docs/source/terms_and_model.rst

```diff
@@ -862,7 +863,7 @@ subclasses, but are still treated as variation.
   implementation SHOULD persist the original Text object and respond
   to queries matching the Text object with the new object.
 * Additional Variation subclasses are continually under
-  consideration. Please open a `GitHub issue`_ if you would like to
+  consideration. Please open a `github issue <github-issues>`_ if you would like to
```

Comment:
On readthe docs, the link to github issue, is to https://vr-spec.readthedocs.io/en/186-haplotype/github-issues, and "Page NotF ound"

Reply:
Fixed. Good catch!